.circleci/continue_config.yml

```diff
@@ -484,7 +494,7 @@ workflows:
           matrix:
             parameters:
               subproject: [ age_estimation, automatic_speech_recognition, classification, keypoint_detection, question_answering, rain_forecast, semantic_segmentation, speaker_diarization, object_detection_2d, semantic_textual_similarity, person_detection, crossing_pedestrian_detection, named_entity_recognition ]
-              resource-class: [ small ]
+              resource-class: [ medium ]
```

Comment:
Let's revert this as soon as https://github.com/kolenaIO/kolena/pull/683 merges. Also worth adding a comment saying so, assuming we can do it without an extra CI trigger.